cspell.json

```diff
@@ -1,46 +1,115 @@
 {
     "version": "0.1",
     "enableFiletypes": [
-        "git-rebase",
-        "pip-requirements"
+        "git-commit",
+        "jupyter"
     ],
     "flagWords": [
         "colour",
+        "favour",
+        "flavour",
         "hte",
         "optimise"
     ],
     "ignorePaths": [
+        "*.rst_t",
+        ".gitignore",
+        ".pre-commit-config.yaml",
+        ".pylintrc",
+        ".readthedocs.yml",
+        ".travis.yml",
+        ".vscode/*",
+        ".vscode/.gitignore",
+        "Makefile",
+        "codecov.yml",
         "cspell.json",
-        ".vscode/*"
+        "doc/_templates/*",
+        "doc/conf.py",
+        "examples/workflow/1_create_model.ipynb",
+        "pyproject.toml",
+        "requirements*.txt",
+        "setup.cfg",
+        "setup.py",
+        "tox.ini"
     ],
     "language": "en-US",
     "words": [
+        "blatt",
+        "bottomness",
+        "breit",
+        "charmness",
+        "clebsch",
+        "conda",
         "docstrings",
+        "expertsystem",
+        "flatté",
+        "functors",
+        "gell",
+        "gordan",
+        "graphviz",
+        "helicity",
+        "ipynb",
+        "ipython",
+        "isospin",
+        "itertools",
         "jupyter",
         "mypy",
-        "nbstripout",
+        "nishijima",
+        "numpy",
         "permalinks",
+        "pycompwa",
         "pydocstyle",
         "pylint",
-        "pypi",
         "pytest",
-        "repos",
+        "redeboer",
+        "setuptools",
+        "spflueger",
+        "struct",
+        "sympy",
+        "tensorflow",
         "tensorwaves",
-        "xcode"
+        "toctree",
+        "topness",
+        "unbinned",
+        "venv",
+        "weisskopf",
+        "xcode",
+        "zfit"
     ],
     "ignoreWords": [
-        "codecov",
-        "envlist",
-        "htmlcov",
-        "ipynb",
-        "linkcheck",
+        "amplitf",
+        "atfi",
+        "cano",
+        "celltoolbar",
+        "codacy",
+        "codemirror",
+        "cparity",
```

Comment:
Should this be a word?

Reply:
Good question. I added this as one word in the `expertsystem` too, because it is used in the recipe file and is now also starting to become part of the interface of `Particle` (https://github.com/ComPWA/expertsystem/pull/100). So it comes down to a design decision.

Reply:
i guess we should either write `c_parity` in the code and c-parity or c parity in the docs. What do you think?

Reply:
`gparity` is in there as well btw

Reply:
Ok can do. Seems that [wiki says](https://en.wikipedia.org/wiki/C_parity) _C_ parity (no dash)

Reply:
Then I'll also fix this in https://github.com/ComPWA/expertsystem/pull/100, comes exactly at the right time.

Reply:
ok good

Reply:
a5707a164ef5dcad2f3e9543974ef0e9af2b6a7a

Reply:
Ok, since the GParity and CParity info is not used in TW this should not cause any problems.